README.md

```diff
@@ -1,42 +1,49 @@
-To use this template create a new repository and select `example-pkg` from the template dropdown. Make sure to name your repository following `<name>-pkg` convention. Make sure to apply branch protection rules to `main` branch and enable `dependabot` for security updates.
-If you're interested in automated dependency updates make sure that `renovate` has access to the new repository.
-
-# Things to do change when you use this repository as a template:
-- [ ] replace `example-pkg` with the name of your package in this `README`
-- [ ] update the `@smg-automotive/example` with the name of your package in `package.json`
-- [ ] update `repository` and `homepage` sections in `package.json` to point to your GitHub repository
-- [ ] ensure that Frontend team has the admin access to the repository
-- [ ] ensure that the Bots team has write access to the repository (this is needed to release the package)
-- [ ] provide usage examples in the `README.md`
-- [ ] enable the project on circleci.com to build and test your package
-- [ ] change the circleci status badge in this `README` to the new project
-- [ ] develop an awesome package
-- [ ] live long and prosper
-# example-pkg
-
-[![CircleCI](https://circleci.com/gh/smg-automotive/example-pkg/tree/main.svg?style=svg&circle-token=c183f151fea3c74453cf8dd962d31e115906a300)](https://circleci.com/gh/smg-automotive/example-pkg/tree/main)
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+# SMG-Automotive ESLint Config
 
 ## Usage
-```
-npm install @smg-automotive/example
+
+```shell
+npm install @smg-automotive/eslint-config
 ```
 
-## Development
+Add a `lint` script to `package.json`:
+
 ```
-npm run build
+  "lint": "eslint --ext ts,js,tsx,jsx,json ."
 ```
 
-You can link your local npm package to integrate it with any local project:
+Add a `format` script to `package.json` to be able to use auto fix:
+
 ```
-cd smg-automotive-example-pkg
-npm run build
+  "format": "npm run lint -- --fix",
+```
+
+### ESLint configuration
+
+Create `.eslintrc.js` in the root of the project:
 
-cd <project directory>
-npm link ../smg-automotive-example-pkg
 ```
+  module.exports = {
+    extends: [
+      <eslint_configuration_name>
+    ],
+  }
+```
+
+- For plain TypeScript project, use the default one: `"@smg-automotive/eslint-config"`
+- For React project, use `"@smg-automotive/eslint-config/react"`
+- For Next.js project, use `"@smg-automotive/eslint-config/next"`
+
+### EditorConfig configuration
+
+Create/Update your `.editorconfig` by [this content](https://github.com/smg-automotive/eslint-config-pkg/blob/main/.editorconfig).
```

Comment:
From @masone 

> Is there some sort of postinstall hook in npm where we could copy the file for the user of the library?

This is a base agreement for `.editorconfig`. If we all agree, we can update the content of this file in `example-pkg`, and other projects can start using it right after repository is created

Reply:
@masone I think there are capabilities for either pos-tinstall messages or actions. I wouldn't go in that direction, though. I wouldn't even add `.editorconfig` to the example repos, since this might overwrite personal settings.
One can set `.editorconfig` on the workspace level, instructing how to do that should be enough.

Reply:
It seems like we interpret [discussion in the RFC](https://github.com/smg-automotive/au-docs/discussions/10#discussioncomment-2684546) in different ways. I understood that we would NOT include it in the templates but share it through the package and everyone can configure it/extend their own as they see fit.
Was I off?

Reply:
Jain, one can still have a broader config for things they like, and since they can be repo specific I'd `.gitignore` this on other repos to allow customisation. But that's my personal preference - if it's easier for the team to have it tin the templates - let's roll with this.